network/src/lib.rs

```diff
@@ -80,6 +80,14 @@ mod cost {
 	pub(super) const UNKNOWN_PEER: i32 = -50;
 	pub(super) const COLLATOR_ALREADY_KNOWN: i32 = -100;
 	pub(super) const BAD_COLLATION: i32 = -1000;
+
+
+	pub(super) const EXPECTED_MESSAGE: i32 = 20;
+	pub(super) const VALID_FORMAT: i32 = 20;
```

Comment:
the pattern we've used elsewhere is to have another module `benefit` for beneficial actions.